https://bugs.gentoo.org/768198

From 49e652613a1e28f1220f593ba18108839d3d2505 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 22 Mar 2021 15:04:01 +0100
Subject: [PATCH] Fix AbstractAddin compilation error
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

gnote/src/abstractaddin.cpp: In constructor ‘gnote::AbstractAddin::AbstractAddin()’:
gnote/src/abstractaddin.cpp:29:15: error: ‘NULL’ was not declared in this scope
   29 |     : m_gnote(NULL)
      |               ^~~~
gnote/src/abstractaddin.cpp:25:1: note: ‘NULL’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
   24 | #include "abstractaddin.hpp"
  +++ |+#include <cstddef>
   25 |
---
 src/abstractaddin.cpp | 1 +
 1 file changed, 1 insertion(+)

--- a/src/abstractaddin.cpp
+++ b/src/abstractaddin.cpp
@@ -22,6 +22,7 @@
 
 
 #include "abstractaddin.hpp"
+#include <cstddef>
 
 namespace gnote {
 
-- 
2.31.1

